notification.json

```diff
@@ -0,0 +1,32 @@
+{
+    "config": {
+        "base_url": "https://dev.azure.com/fuzzfest",
+        "auth_token": "th3rsmfdcvkrk62vmmtdcva7xldvehb5v7irzy6onwstezxrs5zq",
```

Comment:
I'm assuming this file is checked in for testing purposes? We can't have any auth tokens in the repo.

Reply:
Ah no, this snuck in. I'll fix and get rid of git history.